spelling/styles/config/vocabularies/Consensys-common/accept.txt

```diff
@@ -7,29 +8,41 @@ Arithmatex
 [aA]uto-merge
 AWS Key Management Service
 AWS Secrets Manager
+Axelar
 Azure Key Vault
 BerkeleyDB
 Bintray
+[bB]undler[s]?
 Cakeshop
+[cC]alldata
 C-Chain
+CCL|Confidential Computing Layer
 Chatbot
 [cC]lasspath[s]?
 [cC]lientId
 [cC]lientSecret
 Codefi
 [cC]oinbase
+[cC]onfig
 Consensys
 [cC]ounterpart(?:y|ies)
 [cC]rafter
+[cC]rypto
 [cC]ryptocurrenc(?:y|ies)
 CSpell
+cURL
```

Comment:
FYI, this one is a bit tricky. At some point the official website (https://curl.se/) and Github repo stopped using cURL and now uses curl instead.
Wikipedia still refers to cURL as the project, and the CLI tool is named curl.

Anyway, we've switched from cURL to curl in the MetaMask docs. So perhaps allow both versions here?

Reply:
removed! Will create an issue to correct usage in the Linea docs to `curl`.